src/opdsrequestmanager.cpp

```diff
@@ -14,9 +15,23 @@ void OpdsRequestManager::doUpdate(const QString& currentLanguage, const QString&
         query.addQueryItem("lang", currentLanguage);
     }
     query.addQueryItem("count", QString::number(0));
-    if (categoryFilter != "all") {
+    if (categoryFilter != "all" && categoryFilter != "other") {
         query.addQueryItem("tag", categoryFilter);
     }
+
+    if (categoryFilter == "other") {
+        auto categoryList = KiwixApp::instance()->getMainWindow()->getSideContentManager()->getCategoryList();
+        QString excludeTags;
+        for (int i = 0; i < categoryList.size(); i++) {
+            if (categoryList[i] != "Other") {
+                excludeTags += (i != 0) ? ";" : "";
+                excludeTags += categoryList[i].toLower();
+            }
+        }
```

Comment:
`QStringList` (the type of categoryList) already has a function to join string together (https://doc.qt.io/qt-5/qstringlist.html#join)

`excludeTags = categoryList.join(";");` should work